contrib/pkg/createcluster/create.go

```diff
@@ -79,6 +79,7 @@ const (
 	deleteAfterAnnotation      = "hive.openshift.io/delete-after"
 	tryInstallOnceAnnotation   = "hive.openshift.io/try-install-once"
 	tryUninstallOnceAnnotation = "hive.openshift.io/try-uninstall-once"
+	hiveutilCreatedLabel       = "hive.openshift.io/hiveutil-created"
```

Comment:
I need something to match when creating selectorsyncsets so added this label.

Reply:
Good idea.